PrivacyPolicy.md

```diff
@@ -0,0 +1,31 @@
+# Information collected and transmitted by Git Extensions
+
+We would like your permission to collect a little information about how you use Git Extensions. This will help us to make the application better.
+You can opt-out any time.
+
+First, a reminder: Git Extensions is provided "as is", without warranty of any kind, express or implied, including but not limited to the warranties of merchantability, fitness for a particular purpose and noninfringement.
+
+With that out of the way, here's a breakdown of all the information we may collect via Application Insights.
+
+### Application-Level
+Includes:
+* Exception information (could, in rare cases, contain paths on your computer)
+* App version number (e.g. 2.0.x.x)
+* Is portable version
+* Build type (whether the application is an official release build or not)
+* Selected layout settings (such as visibility of the left panel, commit info position etc)
+* Change of selected layout settings
+* Git version (e.g. 2.19.0)
+* SSH client (e.g. OpenSSH or PuTTY)
```

Comment:
If SSH logging is changed:
SSH client path (e.g. OpenSSH or PuTTY)

Reply:
Done
![image](https://user-images.githubusercontent.com/4403806/60593937-24af8680-9de7-11e9-86bd-8affc67de063.png)